plugins/dynamix/FTP.page

```diff
@@ -16,12 +16,38 @@ Tag="globe"
  */
 ?>
 <?
-$ftp_userlist_file = "/boot/config/vsftpd.user_list";
+// Don't need this file any more - IMTheNachoMan on 2021-09-16
+// $ftp_userlist_file = "/boot/config/vsftpd.user_list";
+
+// Where we'll save what each user's chroot jail is - IMTheNachoMan on 2021-09-16
+$user_config_dir = "/boot/config/vsftpd.user_config_dir";
+
 $ftp_userlist = "";
-if (file_exists($ftp_userlist_file)) {
-  $ftp_userlist = str_replace("\n", " ", trim(file_get_contents($ftp_userlist_file)));
-  if ($ftp_userlist === false) $ftp_userlist = "";
+// We can't just get the conents of a file. We have to iterate through the files in a directory - IMTheNachoMan on 2021-09-16
```

Comment:
Old code should be removed, git will preserve the history.

Reply:
Doh. Forgot to remove personal notes/comments. Thanks!